doc/setup.md

```diff
@@ -148,13 +148,19 @@ setting `debug_login` to `False`.
 ### Creating an administrative user
 
 By default, no user has permission to perform system administration.
-You can add a user with such permissions by running the following from
-Python:
+You can give such permissions to an existing user by running the following from your terminal in the `skyportal` directory:
+```
+PYTHONPATH=. python tools/elevate_user.py --username="<username>" --role="<role>"
```

Comment:
Can we change the name of the script to something a bit more descriptive, like `set_user_role`?  I think `elevate` always means up!